### ISSUE TITLE
Crash when overscroll on iOS

### ISSUE THREAD

**Issue by konDeichmann** [MarcinusX/NumberPicker#12]
When there is a minValue of 3 and a maxValue of 10 (for example) and you overscroll the min or the max you will have a crash on iOS.

Stack trace:

```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following assertion was thrown building NumberPickerDialog(dirty, state:
_NumberPickerDialogControllerState#266d1):
'package:numberpicker/numberpicker.dart': Failed assertion: line 31 pos 16: 'initialValue >=
minValue && initialValue <= maxValue': is not true.
Either the assertion indicates an error in the framework itself, or we should provide substantially
more information in this error message to help you determine and fix the underlying cause.
In either case, please report this assertion by filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new
When the exception was thrown, this was the stack:
#2      new NumberPicker.integer (package:numberpicker/numberpicker.dart)
#3      _NumberPickerDialogControllerState._buildNumberPicker (package:numberpicker/numberpicker.dart:383:18)
#4      _NumberPickerDialogControllerState.build (package:numberpicker/numberpicker.dart:397:16)
#5      StatefulElement.build (package:flutter/src/widgets/framework.dart:3730:27)
#6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3642:15)
#7      Element.rebuild (package:flutter/src/widgets/framework.dart:3495:5)
#8      BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2242:33)
```

**Comment by MarcinusX** [comment]
Hey, thanks for adding this issue!
Would you mind running example app from the current master branch and see if you can still reproduce it?
If it's ok, I will update pub package with the fix :)

**Comment by konDeichmann** [comment]
Hi,
works on master.

Thank you

**Comment by MarcinusX** [comment]
I've published the fix in version 0.1.2.
I'm closing this issue, feel free to reopen if it happens again.
